README.md

```diff
@@ -15,7 +15,7 @@ Quick Start
 ### Engine
 
 A core concept of the `stats` package is the `Engine`. Every program importing
-the package gets a default engine where all metrics produced are aggregated.  
+the package gets a default engine where all metrics produced are aggregated.
```

Comment:
Was this auto-fixed by your editor? The trailing double-space at the end of a line is a newline marker in github's markdown syntax.